main/src/library/Int32.flix

```diff
@@ -364,6 +374,11 @@ namespace Int32 {
         let maxi32 = Int16.toInt32(maximum);
         valueOf(clamp(x, mini32, maxi32)).shortValue() as & Pure
 
+    ///
```

Comment:
Helper function should be right after the function itself.

(Once we have letrec, we will want to inline such helper functions, and having them close will be a useful aid).